### PR TITLE
cyberchef: init at 9.37.3

### PR DESCRIPTION
**Pull request by sebastianblunt**
###### Description of changes

Init latest version of [cyberchef](https://github.com/gchq/CyberChef). The output consists of static files put into share/cyberchef, allowing you to run cyberchef from a local copy instead of the online version that includes tracking.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`) (ran in generated static files in browser)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Related to https://github.com/NixOS/nixpkgs/issues/81418

**Comment by sebastianblunt** [comment]
This writes the html files into `share/cyberchef`, I've noticed there are some packages that use `share/webapps`, if that's a standard I could change it to `share/webapps/cyberchef`.

**Comment by sebastianblunt** [comment]
I know there's been some discussion around whether using node2nix in nixpkgs is good or not, if preferred I can change it to just pull the compiled release instead of compiling it with node2nix from source.

File: pkgs/tools/misc/cyberchef/default.nix [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{ lib, pkgs, fetchFromGitHub, stdenv, ... }:
+
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/cyberchef/default.nix:2]: Don't use pkgs as an input.

File: pkgs/tools/misc/cyberchef/generate.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix jq gnutar gzip curl moreutils
+set -euo pipefail
+
+VERSION=9.37.3
+
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/cyberchef/generate.sh:6]: That will not work with the auto update bot.

File: pkgs/tools/misc/cyberchef/default.nix (hunk 1 above)
Comment (sebastianblunt) [reply pkgs/tools/misc/cyberchef/default.nix:2]: I don't think it's possible to use node2nix without giving it pkgs.

File: pkgs/tools/misc/cyberchef/generate.sh (hunk 2 above)
Comment (sebastianblunt) [reply pkgs/tools/misc/cyberchef/generate.sh:6]: Yes, unfortunately.

Comment (SuperSandro2000) [reply pkgs/tools/misc/cyberchef/generate.sh:6]: Can we make it work though?

Comment (sebastianblunt) [reply pkgs/tools/misc/cyberchef/generate.sh]: Ah, I didn't know the bot could run scripts like that.

It should work now, when running locally it produces a reasonable diff and then errors on `fatal: path 'pkgs/tools/misc/cyberchef/default.nix' exists on disk, but not in 'remotes/upstream/master'`

**Pull request closed without merging**